config/jest.config.js

```diff
@@ -1,44 +1,32 @@
 module.exports = {
     collectCoverage: true,
-    preset: 'ts-jest/presets/js-with-ts',
-    rootDir: '../',
+    preset: "ts-jest/presets/js-with-ts",
```

Comment:
if you use https://www.npmjs.com/package/@guardian/prettier too these will all revert to `'`